mpt/impl/managesieve/core/pom.xml

```diff
@@ -52,8 +52,8 @@
             <artifactId>protocols-managesieve</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
```

Comment:
why do you need testcontainers here?

Reply:
it's a rebase bug, I have to change that.